src/List/NonEmpty.elm

```diff
@@ -175,18 +175,130 @@ product =
 
 
 
---
--- append
--- concat
--- concatMap
--- intersperse
--- map2
--- andMap
---
--- sort
--- sortBy
--- sortWith
--- isSingleton
+-- append ("1", ["2, 3"]) ("4", ["5"])
+
+
+append : NonEmptyList a -> NonEmptyList a -> NonEmptyList a
+append ne1 ne2 =
+    case toList ne2 of
+        [] ->
+            ne1
+
+        _ ->
+            foldr cons ne1 ne2
```

Comment:
the thing which I'm not sore with this implementation is order of arguments.

I would expect:

```
> append (0, [1,2,3]) (10, [11,12,13])
(0,[1,2,3,10,11,12,13])
    : NonEmptyList number
```

while this is:

```
> append (0, [1,2,3]) (10, [11,12,13])
(10,[11,12,13,0,1,2,3])
    : NonEmptyList number
```

but perhaps it would be meant to be used with `|>`:

```
> (0, [1,2,3]) |> append (10, [11,12,13])
(0,[1,2,3,10,11,12,13])
    : NonEmptyList number
```

Anyway this is different from `List.append`:

```
> [0,1,2,3] |> List.append [10,11,12,13]
[10,11,12,13,0,1,2,3]

> List.append [0,1,2,3] [10,11,12,13]
[0,1,2,3,10,11,12,13]
    : List number
```

therefor I think we want to change it as:
```suggestion
        _ ->
           Tuple.mapSecond (\xs -> xs ++ toList ne2) ne1
```
Or you can deconstruct the fist `NonEmptyList` if you don't want to use mapSecond.